problem2.cpp

```diff
@@ -15,15 +15,15 @@ using namespace std;
 class Solution {
     public:
         int change(int amount, int[] coins) {
-            
-        int dp[coins.length+1][amount+1];
+
+        int dp[coins.size()+1][amount+1];
 
         dp[0][0] = 1;
 
-        for(int i =1; i<coins.length+1; i++)
+        for(int i =1; i<coins.size()+1; i++)
             dp[i][0] = 1;
 
-        for(int i =1; i<coins.length+1;i++)
+        for(int i =1; i<coins.size()+1;i++)
```

Comment:
Why did you replace .length by .size() ?

Reply:
I was looking at a Java code for reference and I forgot to change the .length syntax of Java to .size() syntax of c++ while pushing the previous commit.